aten/src/ATen/native/mps/operations/FourierTransform.mm

```diff
@@ -0,0 +1,196 @@
+//  Copyright © 2023 Apple Inc.
+#define TORCH_ASSERT_ONLY_METHOD_OPERATORS
+#include <ATen/native/UpSample.h>
+#include <ATen/native/mps/MPSGraphVenturaOps.h>
+#include <ATen/native/mps/MPSGraphSonomaOps.h>
+#include <ATen/native/mps/OperationUtils.h>
+
+#include <MetalPerformanceShadersGraph/MPSGraph.h>
+#include <MetalPerformanceShadersGraph/MPSGraphFourierTransformOps.h>
```

Comment:
You don't need this one here, do you (as you redefine ops in MPSGraphSonomaOps, don't you?
```suggestion
```